packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -220,43 +221,76 @@ class SnackBar extends StatelessWidget {
     } else {
       children.add(const SizedBox(width: _kSnackBarPadding));
     }
-    final CurvedAnimation heightAnimation = CurvedAnimation(parent: animation, curve: _snackBarHeightCurve);
-    final CurvedAnimation fadeAnimation = CurvedAnimation(parent: animation, curve: _snackBarFadeCurve, reverseCurve: const Threshold(0.0));
-    Widget snackbar = SafeArea(
+    final CurvedAnimation heightAnimation =
+        CurvedAnimation(parent: animation, curve: _snackBarHeightCurve);
+    final CurvedAnimation fadeInAnimation =
+        CurvedAnimation(parent: animation, curve: _snackBarFadeInCurve);
+    final CurvedAnimation fadeOutAnimation = CurvedAnimation(
+        parent: animation,
+        curve: _snackBarFadeOutCurve,
+        reverseCurve: const Threshold(0.0));
```

Comment:
style here doesn't match usual flutter style (see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo)